src/main/java/App.java

```diff
@@ -54,6 +62,20 @@ public void run() {
      * @throws ClassNotFoundException in case JDBC in not found.
      */
     public static void main(String[] args) throws SQLException, ClassNotFoundException {
+        //init each module
+        TrackModel.init();
+        WaysideController.init();
+        CTCModel.init();
+        TrainTracker.getTrainTracker();
+        TrainController.initUI();
+        
+        //open the main gui
         run();
+        
+        //the main update loop
+        while(true){
```

Comment:
max time limit?

Reply:
Is there a reason we would want to stop besides hitting max int?

Reply:
probably just to avoid infinite loops. seems just like good practice, and repeatable termination states. no concrete reason